### PR TITLE
Update blend-modes.md

### PR DESCRIPTION
**Pull request by shinoryo1216**
Fix the inconsistency.

Screenshot_20211227_010702

Screenshot_20211227_011118

### CODE REVIEW

**Comment by elstoc** [comment]
Thanks. I'll merge this when we start 4.0 work.

**Comment by shinoryo1216** [comment]
Another fix.
Screenshot_20211228_141914

Screenshot_20211228_141802

File: content/darkroom/masking-and-blending/blend-modes.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ HSV color
 lightness
 : Mix lightness from the input and output images, while taking color (chroma and hue) unaltered from the input image.
 
-chrominance
+chromaticity
 : Mix chrominance from the input and output images, while taking lightness and hue unaltered from the input image. This blend mode uses RGB ratios, divided by a Euclidean norm.
```
Comment (elstoc) [inline content/darkroom/masking-and-blending/blend-modes.md:140]: presumably we also need to change the word chrominance in the definition.

Comment (shinoryo1216) [reply content/darkroom/masking-and-blending/blend-modes.md:140]: Yeah, maybe we need to ask the developer or the author to check it, but I don't know who.

Comment (elstoc) [reply content/darkroom/masking-and-blending/blend-modes.md:140]: @aurelienpierre says chromaticity is "not wrong" so let's go with that. I'll merge and then copy-edit after.

**Pull request merged**